html/components/footer.stories.js

```diff
@@ -264,12 +264,12 @@ export const defaultStory = () => {
 
             <div class="sprk-o-Stack__item">
               <div data-sprk-toggle="container">
-                <a class="sprk-b-TypeBodyFour sprk-b-Link sprk-b-Link--simple sprk-b-Link--has-icon sprk-u-FontWeight--normal" data-sprk-toggle="trigger" href="#nogo">
+                <button class="sprk-b-TypeBodyFour sprk-b-Link sprk-b-Link--simple sprk-b-Link--has-icon sprk-u-FontWeight--normal sprk-c-Toggle__trigger" data-sprk-toggle="trigger">
```

Comment:
is it supposed to have all these other classes?

Reply:
I think it should still have all of these classes. Semantically this is a button, but stylistically it has not changed. Each of these classes is applying styles that should still be applied.

Reply:
The new toggle__trigger class applies all the same styles as these extra classes. So we won't need them anymore.

Reply:
Gotcha. I was mistaken about what RV's change is.